server/tests/conftest.py

```diff
@@ -1,19 +1,34 @@
 import logging
+from pathlib import Path
+
 import pytest
-import os
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
 
-import connexion
+from athenian.api import AthenianApp
+from athenian.api.models.db import hack_sqlite_arrays
+from athenian.api.models.db.github import Base
+from .sample_db_data import fill_session
 
 
 @pytest.fixture
-def client(loop, aiohttp_client):
+def client(loop, aiohttp_client, sqlite_db):
```

Comment:
I wold make it possible to test against existing db (mainly postgres) as well.

Reply:
+1

Reply:
https://github.com/athenianco/athenian-api/issues/5

Reply:
And this re-opens the issue of opening tickets on JIRA vs GH :p

Reply:
I heard that they are automatically mirrored, no?

Reply:
Oh if that's the case then that's great, I didn't know whether it was already working. \cc @marnovo